tmuxp/workspacebuilder.py

```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf8 -*-
+# -*- coding: utf-8 -*-
```

Comment:
Nice find

Reply:
That was flake8 and the fact that my pull request reduced the coverage. This file came up as the main offender and this was the first thing flake8 pointed out. I figured I would try and update the pull request and see if that improved coverage results, which it did.